independent-publisher-connectors/Electricity Maps/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1239 @@
+                            "x-ms-summary": "Gas"
+                          },
+                          "hydro": {
+                            "type": "number",
+                            "description": "Hydro power consumption forecast",
+                            "x-ms-summary": "Hydro"
+                          },
+                          "nuclear": {
+                            "type": "number",
+                            "description": "Nuclear power consumption forecast",
+                            "x-ms-summary": "Nuclear"
+                          },
+                          "solar": {
+                            "type": "number",
+                            "description": "Solar power consumption forecast",
+                            "x-ms-summary": "Solar"
+                          },
+                          "unknown": {
+                            "type": "number",
+                            "description": "Unknown power consumption forecast",
+                            "x-ms-summary": "Unknown"
+                          },
+                          "wind": {
+                            "type": "number",
+                            "description": "Wind power consumption forecast",
+                            "x-ms-summary": "Wind"
+                          }
+                        },
+                        "type": "object",
+                        "description": "Power Consumption Breakdown data",
+                        "x-ms-summary": "Power Consumption Breakdown"
+                      },
+                      "powerConsumptionTotal": {
+                        "type": "number",
+                        "description": "Power Consumption Total Forecast",
+                        "x-ms-summary": "Total Consumption"
+                      }
+                    },
+                    "type": "object"
+                  },
+                  "type": "array",
+                  "description": "Power Consumption forecast data",
+                  "x-ms-summary": "Power Consumption Forecast"
+                },
+                "updatedAt": {
+                  "type": "string",
+                  "description": "Date and time when forecast has been updated",
+                  "x-ms-summary": "Updated At"
+                },
+                "zone": {
+                  "type": "string",
+                  "description": "Country or Zone",
+                  "x-ms-summary": "Zone"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "operationId": "ConsumptionForecast",
+        "summary": "Get Power Consumption Forecast",
+        "description": "Retrieves the forecasted power consumption breakdown of an area, which represents the physical origin of electricity broken down by production type. It can either be queried by zone identifier or by geolocation"
```

Comment:
@VitaliiSorokin all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)